src/topology/multicodec-topology.js

```diff
@@ -55,43 +55,44 @@ class MulticodecTopology extends Topology {
 
   /**
    * Update topology.
-   * @param {Array<PeerInfo>} peerInfoIterable
+   * @param {Array<{id: PeerId, multiaddrs: Array<Multiaddr>}>} peerDataIterable
```

Comment:
I missed protocols in the last suggestion which is the one we care about
```suggestion
   * @param {Array<{id: PeerId, multiaddrs: Array<Multiaddr>, protocols: Array<string>}>} peerDataIterable
```